examples/mobile/UIComponents/components/notification-badge/sub-tabs-four.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../lib/tau/mobile/js/tau.js"></script>
+	<title>SubTab Four</title>
+</head>
+
+<body>
+	<div class="ui-page">
+		<header>
+			<div class="ui-appbar-title-container">
+				<span class="ui-appbar-title">Sample Dot Badge</span>
+			</div>
+			<div class="ui-appbar-left-icons-container">
+				<a href="#" class="ui-btn ui-btn-icon ui-btn-icon-back" data-style="flat" data-rel="back"></a>
+			</div>
+			<div class="ui-appbar-action-buttons-container">
+				<button class="ui-btn ui-btn-icon ui-btn-icon-more" data-style="flat">
+					<span class="ui-badge"></span>
+				</button>
+			</div>
+		</header>
+		<div class="ui-tabs">
+			<div class="ui-sub-tab ui-sub-tab-static">
+				<ul>
+					<li class="ui-li-anchor">
+						<a class="ui-tab-active" href="#">
+							One
+						</a>
+					</li>
+					<li class="ui-li-anchor">
+						<a href="#">
+							Two
+						</a>
+					</li>
+					<li class="ui-li-anchor">
+						<a href="#">
+							Three
+						</a>
+					</li>
+					<li class="ui-li-anchor">
+						<a href="#">
+							Four
+							<span class="ui-badge"></span>
```

Comment:
ditto.

Reply:
We don't need title value because this is only `dot`.